src/LibMtpSharpStandardMacOS/OpenedMtpDevice.cs

```diff
@@ -134,6 +142,15 @@ public void CopyAzwFileToDevice(string filePath, uint storageId, uint parentId)
                 throw new CopyFileToDeviceException($"Cannot copy {fileInfo.Name} to device");
         }
 
+        public void CopyFileFromDevice(uint fileId, string destinationPath)
+        {
+            if (string.IsNullOrEmpty(destinationPath))
```

Comment:
I think that when we don't have just simple check for null, then `string.IsNullOrWhitespace` is better because it checks also for mistakes likes empty spaces etc.

Reply:
Done